WMFData/Sources/WMFData/Data Controllers/Year In Review/WMFYearInReviewDataController.swift

```diff
@@ -407,54 +401,16 @@ import CoreData
         }
 
         cdReport.year = Int32(year)
+
         if (cdReport.slides?.count ?? 0) == 0 {
             cdReport.slides = try self.initialSlides(year: year, moc: backgroundContext) as NSSet
         }
-        
-        // If needed: Populate initial saveCount slide
-        if var slides = cdReport.slides as? Set<CDYearInReviewSlide> {
-            let containsSaveCount = slides.contains(where: { slide in
-                slide.id == WMFYearInReviewPersonalizedSlideID.saveCount.rawValue
-            })
-            if !containsSaveCount,
-                let initialSaveCountSlide = try? initialSaveCountSlide(year: year, moc: backgroundContext) {
-                slides.insert(initialSaveCountSlide)
-                cdReport.slides = slides as NSSet
-            }
-        }
 
-        // If needed: Populate initial mostReadDay slide
-        if var slides = cdReport.slides as? Set<CDYearInReviewSlide> {
-            let containsMostRead = slides.contains(where: { slide in
-                slide.id == WMFYearInReviewPersonalizedSlideID.mostReadDay.rawValue
-            })
-            if !containsMostRead,
-                let initialMostReadDaySlide = try? initialMostReadDaySlide(year: year, moc: backgroundContext) {
-                slides.insert(initialMostReadDaySlide)
-                cdReport.slides = slides as NSSet
-            }
-        }
-        
-        // If needed: Populate initial viewCount slide
-        if var slides = cdReport.slides as? Set<CDYearInReviewSlide> {
-            let containsViewCount = slides.contains(where: { slide in
-                slide.id == WMFYearInReviewPersonalizedSlideID.viewCount.rawValue
-            })
-            if !containsViewCount,
-                let initialViewCountSlide = try? initialViewCountSlide(year: year, moc: backgroundContext) {
-                slides.insert(initialViewCountSlide)
-                cdReport.slides = slides as NSSet
-            }
-        }
-        
         try self.coreDataStore.saveIfNeeded(moc: backgroundContext)
 
         guard let iosFeatureConfig = developerSettingsDataController.loadFeatureConfig()?.ios.first,
-              let yirConfig = iosFeatureConfig.yir(yearID: targetConfigYearID) else {
-            return nil
-        }
-
-        guard let cdSlides = cdReport.slides as? Set<CDYearInReviewSlide> else {
+              let yirConfig = iosFeatureConfig.yir(yearID: targetConfigYearID),
```

Comment:
I'm against commenting code because it can lead to us forgetting about this (since it won't execute) or being untested.
My suggestion is:

On the data controller, wrap the remote config in a check like this.
var yirConfig: Bool = false
#if DEBUG
yirConfig = iosFeatureConfig.yir(yearID: targetConfigYearID) // so we still can test it in dev
#endif

This way it'll only see it in debug mode.